pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### What issue is this solving?
+Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
+
+### Any helpful knowledge/context for the reviewer?
+Is a `npm install` necessary?
+Any special requirements to test?
+
+### Please make sure you've attempted to meet the following coding standards
+- [ ] Code builds successfully
+- [ ] Code has been tested and does not produce errors
+- [ ] Code is readable and formatted
+- [ ] There isn't any unnecessary commented-out code
+- [ ] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)
```

Comment:
After this checklist we might want to put a friendly disclaimer like, "If you're having trouble meeting this criteria reach out on slack and we'll help!"

Reply:
I like that! I'll add it :)